client/Makefile

```diff
@@ -15,8 +17,6 @@ GO_PACKAGES_REPO_PATH = $(addprefix $(repo_path)/,$(GO_PACKAGES))
 GOFMT = gofmt -e -l -s
 GOTEST = go test --cover --race -v
 
-VERSION := $(shell git describe --tags --abbrev=0 2>/dev/null)+$(shell git rev-parse --short HEAD)
```

Comment:
Note that the `VERSION` in client vs workflow are different. This one combines two git outputs